src/FastBoard.cpp

```diff
@@ -255,17 +268,30 @@ bool FastBoard::is_suicide(int i, int color) {
     }
 }
 
+/*
+    Liberties on a single point - called pseudo liberties.
```

Comment:
If you're going to add an explanation of pseudoliberties, they should probably go with the respective datastructure in the header, rather than a random function using them. (The explanation is also not very clear, I think)

Reply:
I was basing the comment on what @fishcu said, and which you indicated was a correct description. There is no data structure for pseudoliberties that I can find - only the count_pliberties method which calls count_neighbors(EMPTY, i). I reduced the comment to just a single line until a better description can be found.